less/index.less

```diff
@@ -49,11 +49,214 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
-* {
-    box-sizing: border-box;
+
+// variables 
+
+@font-stack: 'Roboto', sans-serif;
+@base-font-size: 62.5%;
+@font-color: black;
+@mobile: (max-width: 515px);
+
+// mixins
+
+.custom-font-style(@size, @weight) {
+	font-size: @size;
+	font-weight: @weight;
+	color: black;
+}
+
+.column-center() {
+	display: flex;
+	justify-content: center;
+	flex-direction: column;
+	align-items: center;
+}
+
+.progress-bar-layout(@pbw) {
+	background-color: white;
+		  border-radius: 13px;
+		  border: .45px #686767 solid;
+		  padding: 2px;
+		  width: 60%;
+		  margin: 0 0 0 20px;
+	&>div {
```

Comment:
nice job using & and > selector before learning about it!

Reply:
Thanks, Ben!